cob_obstacle_distance/src/distance_manager.cpp

```diff
@@ -90,7 +90,7 @@ int DistanceManager::init()
         return -1;
     }
 
-    if(!nh_.getParam("joint_names", this->joints_))
+    if(!nh_.getParam("/arm_left/joint_names", this->joints_))
```

Comment:
do not use absolute namespaces! With this the distance_manager is just usable with the left arm...